zkvm/src/types.rs

```diff
@@ -13,6 +13,9 @@ use crate::ops::Instruction;
 use crate::predicate::Predicate;
 use crate::transcript::TranscriptProtocol;
 
+use std::ops::Add;
```

Comment:
can do these in one line:
```suggestion
use std::ops::{Neg,Add};
```